addons/web/static/src/core/notifications/notification.xml

```diff
@@ -2,14 +2,16 @@
 <templates xml:space="preserve">
 
     <t t-name="web.NotificationWowl" owl="1">
-        <div t-attf-class="o_notification {{className}}"
+        <div t-attf-class="o_notification overflow-hidden bg-white"
         role="alert" aria-live="assertive" aria-atomic="true">
-            <div class="o_notification_body">
-                <button type="button" class="close o_notification_close"
+            <div t-attf-class="toast-header px-3 py-2 border border-{{className}} border-right-0 border-top-0 border-bottom-0">
```

Comment:
Since you have to define a custom border anyway (meaning 1 scss line), this group of border's classes are a bit overkill.
Read more in the related SCSS comment